models/labels/dao/labels_dao.sql

```diff
@@ -0,0 +1,45 @@
+{{config(alias='dao')}}
+
+SELECT 
+    array(blockchain) as blockchain,
+    dao as address, 
```

Comment:
qq @henrystats I'm not sure I understand the difference between the `dao` and `dao_wallet_address` columns here ?

Reply:
hola @soispoke, so, for example when you create a DAO using Aragon, you also need to create an APP (a smart contract) that would receive and send funds that belong to that DAO and for other actions too. 

For DAO Haus, the moloch (the DAO) receives and sends funds, but a moloch can also set up a minion (a gnosis safe with reality.eth module) that can be used by the DAO to receive and send funds.

For colony, it's the DAO address that receives and sends fund. So on the dao.addresses table, there are multiple entries of **DAO** but the dao wallet address table is different (daos can have multiple wallets, e.g with Aragon).

That's why I'm selecting both dao & dao_wallet_address since they belong to a DAO.